services/intersection-api/Dockerfile

```
@@ -59,37 +78,41 @@ RUN mvn -s settings.xml dependency:resolve
 COPY ./api/src ./src
 RUN mvn -s settings.xml install -DskipTests
 
-# # === RUNTIME IMAGE for Java and ACM ===
-# # Use Amazon Corretto Java on Amazon Linum 2023 to match the codec build env
-# # FROM alpine:3.12
-FROM amazoncorretto:21-al2023
+# === RUNTIME IMAGE for Java and ACM ===
+# NOTE: This is alpine based to match the ACM image and allow for ARM64 support.
+FROM eclipse-temurin:21-alpine
 
 WORKDIR /home
+USER root
 
+# Install runtime dependencies
+RUN apk update && apk add --no-cache \
+    bash \
+    libstdc++ \
+    zlib \
+    openssl \
+    curl \
+    librdkafka \
+    librdkafka-dev \
+    jemalloc
```

Suggesttion: remove unnecessary dependencies from runtime:
```suggestion
RUN apk update && apk add --no-cache \
    bash \
    librdkafka \
    librdkafka-dev \
    jemalloc
```